installkernel

```diff
@@ -78,9 +78,16 @@ updatever() {
 # into kernel package installation.  Also make sure the PATH includes
 # /usr/sbin and /sbin, just as dpkg would.
 if [ "${dir}" = "/boot" ]; then
-	PATH="${PATH}:/usr/sbin:/sbin" \
-		run-parts --verbose --exit-on-error --arg="${ver}" --arg="${img}" \
-		/etc/kernel/preinst.d
+	(
+		export LC_ALL=C PATH="${PATH}:/usr/sbin:/sbin"
+
+
+		for x in /etc/kernel/preinst.d/*; do
```

Comment:
while we are changing things here anyway, lets align this with ˋUSE=+systemdˋ and do ˋ/etc/kernel/preinst.d/*.installˋ instead.

Reply:
Bit worried that users might be relying on the existing behaviour. Probably best to do that separately.